cdk.tf/vercel.json

```diff
@@ -33,6 +33,11 @@
       "destination": "https://github.com/hashicorp/terraform-cdk/issues/:id",
       "permanent": false
     },
+    {
+      "source": "/prs/:id",
```

Comment:
Since this is a team-only thing, let's put this under an `internal` or such path.

Reply:
But that makes the url longer 😄
(And we already have /issues; see directly above)

Reply:
Couldn't you technically reuse that one? All PRs are issues (but not all issues are PRs).
https://cdk.tf/issues/2540

Reply:
Right.. Yeah, I could do that 💡